api/database/migrations/20221108222521-create-user.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+const { DataTypes } = require('sequelize')
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+     await queryInterface.createTable('User', {
+       id:{
+        type:DataTypes.UUID,
+        defaultValue: DataTypes.UUIDV4,
+        unique: true,
+        primaryKey: true,
+        allowNull: false
+       },
+       fullname: {
+        type: DataTypes.STRING,
+        allowNull: false
+       },
+       email: {
+        type: DataTypes.TEXT,
+        allowNull: false
+       },
+       password: {
+        type: DataTypes.STRING,
+        allowNull: false
+       },
+       phone: {
+        type: DataTypes.INTEGER
```

Comment:
change datatype integer to string because for the length of the phone it is not enough

Reply:
resolved